src/elements/Accordion/Accordion.js

```diff
@@ -6,72 +6,160 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import Div from '../Div/Div';
-import Grid from '../Grid/Grid';
+import FlexRow from '../FlexRow/FlexRow';
+import Flex from '../Flex/Flex';
+import Button from '../Button/Button';
 import { isMobile } from '../../util/helpers';
 
+const AccordionCaret = ({ white, up }) => (
+	<object className={classNames('ui-accordion-caret', { white, up })}>
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="-5 -5 26 26" preserveAspectRatio="xMaxYMax meet">
+			<defs>
+				<style>
+					{`
```

Comment:
does this need string interpolation?

Reply:
Yes, the brackets in the CSS break JSX without it